solver/src/main.rs

```diff
@@ -173,7 +192,7 @@ async fn main() {
         price_estimator.clone(),
     );
     let liquidity_collector = LiquidityCollector {
-        uniswap_liquidity,
+        uniswap_like_liquidity: vec![uniswap_liquidity, sushiswap_liquidity],
```

Comment:
Can we decide the contents of this vector based on a config value for now? This way we can turn off specific liquidity without a code change.